sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -397,7 +397,7 @@ dateTimeFunctionName
     : ADDDATE | CONVERT_TZ | DATE | DATETIME | DATE_ADD | DATE_FORMAT | DATE_SUB | DAY | DAYNAME | DAYOFMONTH | DAYOFWEEK
     | DAYOFYEAR | FROM_DAYS | FROM_UNIXTIME | HOUR | MAKEDATE | MAKETIME | MICROSECOND | MINUTE
     | MONTH | MONTHNAME | QUARTER | SECOND | SUBDATE | SYSDATE | TIME | TIME_TO_SEC | TIMESTAMP
-    | TO_DAYS | UNIX_TIMESTAMP | WEEK | YEAR
+    | TO_DAYS | UNIX_TIMESTAMP | UTC_DATE | UTC_TIME | UTC_DATETIME | WEEK | YEAR
```

Comment:
same here, why is it called UTC_DATETIME vs UTC_TIMESTAMP

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/commit/59ef32f37ea15e9f44cae06574c3cb9f27db26e1.